src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2979,7 +2982,7 @@
         "locations": {
           "description": "An array of the artifactLocation objects associated with the tool component.",
           "type": "array",
-          "minItems": 0,
+          "minItems": 1,
```

Comment:
this is not correct, s/be minItems of 0. note the discrepancy between this value and the default value of empty array provided.

Reply:
you are right, the spec does say that it should be 0. Will correct this.

---
In reply to: [276423146](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276423146) [](ancestors = 276423146)